docs/src/examples.md

```diff
@@ -11,7 +11,7 @@ using OrdinaryDiffEq
 rn = @reaction_network begin
     σ, 0 --> A
     d, A --> 0
```

Comment:
You should consider converting these at some point to `@example` blocks in the future, which will dynamically run the code when building the docs, and let you see if there is an issue with it.